### ISSUE TITLE
run: command not found

### ISSUE THREAD

**Issue by m4z** [RocketChat/install.sh#63]
```
# mkdir /opt/$(date +%F)_backup-rocket.chat-db && rocketchatctl backup --backup-dir=/opt/$(date +%F)_backup-rocket.chat-db
/usr/local/bin/rocketchatctl: line 1106: run: command not found
Can't write in the backup directory: /opt/2022-05-02_backup-rocket.chat-db
```

That line is:
```
touch "$backup_dir/testfile_$sufix.txt" /dev/null 2>&1 && run rm "$backup_dir/testfile_$sufix.txt" || print_backup_dir_error_and_exit
```

Also on line 896:
```
run rm -rf $ROCKETCHAT_DIR_UPDATE
```

Also on line 1175:
```
command_exists nvm || { test -s /opt/nvm/nvm.sh && run source $_ || do_install_nvm; }
```

Please fix. Thanks.

**Comment by cmoore-mayesh** [comment]
> Same problem here -- not really understanding why its happening. The specified backup folder exists and has permissions to be written to.
As a temporary work around, I've changed line 1106 to this the following

`touch "$backup_dir/testfile_$sufix.txt" /dev/null 2>&1 || print_backup_dir_error_and_exit`

Backup is in progress now.  it just has one extra testfile in there that can be manually removed.

**Comment by m4z** [comment]
> Same problem here -- not really understanding why its happening.

I believe the problem is that a debug alias ("`run`") was left in the code by mistake. Or that command is from a missing / undefined software dependency.

**Comment by xmedeko** [comment]
Partially fixed by #68
